driver_test.go

```diff
@@ -167,6 +167,16 @@ type DBTest struct {
 	conn *sql.Conn
 }
 
+func (dbt *DBTest) connParams() map[string]*string {
+	var params map[string]*string
+	dbt.conn.Raw(func(driverConn any) error {
```

Comment:
`error` is returned here. Maybe we should catch it and run `assertNilF(dbt.t, err)`?

Reply:
nice catch, added.